pkg/vsphere/vm/vm.go

```diff
@@ -519,20 +519,20 @@ func (vm *VirtualMachine) Properties(ctx context.Context, r types.ManagedObjectR
 	defer trace.End(trace.Begin(fmt.Sprintf("VM(%s) Properties(%s)", r, ps), op))
 
 	contains := false
-	for i := range ps {
-		if ps[i] == "summary" || ps[i] == "summary.runtime" {
+	for _, v := range ps {
+		if strings.HasPrefix(v, "summary") {
```

Comment:
This would also match `summary.config` - and in that circumstance we would not have the Runtime structure populated. I *think* this would result in a panic, but might be worse and just give you a zero initialized structure.
This could have been correctly phrased as:
```
strings.HasPrefix("summary.runtime") || strings.Equals("summary")
```